TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/tests/test_cli.py

```diff
@@ -66,6 +67,35 @@ def test_cli_with_backfill_describe(runner, env, mocker):
     assert result.exit_code == 0
 
 
+def test_cli_snapshot_create(runner, env, mocker):
+    mock_create = mocker.patch('console_link.models.snapshot.Snapshot.create')
```

Comment:
does this work? I think you might have to do a patch object, because `create` isn't a static method

Reply:
Fixed by moving to mock the logic function